packages/esm-patient-forms-app/src/config-schema.ts

```diff
@@ -61,6 +61,11 @@ export const configSchema = {
       },
     ],
   },
+  displayPOCForms: {
```

Comment:
Also, to my dismay, the term AMPATH Forms, has stucky. I think you are specifically referring her to AMPATH forms when using the term POC. Perhaps replace with AF for now.

Reply:
See my requested change above